### PR TITLE
Add a new function Remark and Test cases

### PR DESCRIPTION
**Pull request by Blizzeracz**
Tutorial 2 - Adding a new function Remark. Do not merge!

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/RemarkCommand.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        if (!(other instanceof RemarkCommand)) {
+            return false;
+        }
+
+        RemarkCommand e = (RemarkCommand) other;
+        return index.equals(e.index)
+                && remark.equals(e.remark);
+    }
+
+}
```
Comment (euchangxian) [inline src/main/java/seedu/address/logic/commands/RemarkCommand.java:91]: checkstyle violation here. Files should end with a newline. Please amend this! 😞

File: src/main/java/seedu/address/model/person/Remark.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        return other == this // short circuit if same object
+                || (other instanceof Remark // instanceof handles nulls
+                && value.equals(((Remark) other).value)); // state check
+    }
+
+    @Override
+    public int hashCode() {
+        return value.hashCode();
+    }
+}
```
Comment (euchangxian) [inline src/main/java/seedu/address/model/person/Remark.java:33]: checkstyle violation here! Files should end with a new line

File: src/test/java/seedu/address/logic/commands/RemarkCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+                new Remark(VALID_REMARK_AMY))));
+        // different remark -> returns false
+        assertFalse(standardCommand.equals(new RemarkCommand(INDEX_FIRST_PERSON,
+                new Remark(VALID_REMARK_BOB))));
+    }
+
+    @Test
+    void execute() {
+    }
+}
```
Comment (euchangxian) [inline src/test/java/seedu/address/logic/commands/RemarkCommandTest.java:133]: checkstyle violation here! Files should end with a new line! 😞

**Review by euchangxian — CHANGES_REQUESTED** [review]
UwU, LGTM!!! But oh my fluffy tail, pwease fix those super-duper cheeky checkstywe viowations befowe next time, okay? Teehee >w< 😄 😄 😄✨✨🌸🌈 OwO, keep up the adowable work! You're doing pawsitively purrfect and I believe in you! Keep shining and spreading those uwu vibes! 💖💫🌟

File: src/main/java/seedu/address/logic/commands/RemarkCommand.java (hunk 1 above)
Comment (Blizzeracz) [reply src/main/java/seedu/address/logic/commands/RemarkCommand.java:91]: thank you pookie boo!

**Pull request closed without merging**